From 963e04688a19e125b86ba026d3ef09e4afe17f69 Mon Sep 17 00:00:00 2001
From: Daniel Hahler <git@thequod.de>
Date: Mon, 2 Mar 2015 19:38:22 +0100
Subject: [PATCH] Remove assertion in django_settings_is_configured

django.conf.settings.configured is not necessarily True, in case DSM
wasn't set during `pytest_load_initial_conftests`, but gets set in the
project's conftest (via `pytest_configure`).

Fixes https://github.com/pytest-dev/pytest-django/issues/146
---
 pytest_django/lazy_django.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/pytest_django/lazy_django.py b/pytest_django/lazy_django.py
index 8458040..4ba4d5a 100644
--- a/pytest_django/lazy_django.py
+++ b/pytest_django/lazy_django.py
@@ -22,8 +22,6 @@ def django_settings_is_configured():
 
     # If DJANGO_SETTINGS_MODULE is defined at this point, Django is assumed to
     # always be loaded.
-    from django.conf import settings
-    assert settings.configured is True
     return True
 
 
